script.js

```diff
@@ -5,7 +5,7 @@ class BookCollection {
     this.errorMsg = document.getElementById('error');
```

Comment:
Good job working on some of the project requirements but I noticed your design is missing a few thing that need to be addressed before your work gets an approval. If you carefully look at the template design or wireframe it displays date and time in all page tabs but for your design it is not displaying the date and time. This was supposed to be done in using [luxon](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time)

This is how your work looks:



![Screenshot 2023-06-02 at 7 12 50 AM](https://github.com/kamranasif506/Awesome-books/assets/79506105/9bf3aa8c-fda0-4259-81f7-57d4ff95ffbc)

Please follow this wireframe to display date and time:

![DATE AND TIME ](https://github.com/kamranasif506/Awesome-books/assets/79506105/506a66fe-e959-412a-b33b-def74caa00d1)

Remember this should appear in all three pages of your tabs.



2. Secondly it would be nice to properly align the content in your contact page to properly match the design template you can do this by adjusting margin and padding.

This is your contact page:

![Screenshot 2023-06-02 at 7 12 21 AM](https://github.com/kamranasif506/Awesome-books/assets/79506105/488499da-4ada-4b22-be8e-12e2d8b13041)

your page content should be centered properly.

Please follow this template design:

![Contact page ](https://github.com/kamranasif506/Awesome-books/assets/79506105/528e841b-9bf1-4aaf-8b04-bc5c8861714f)